src/example/webChat/client/src/exampleServiceDesk.ts

```diff
@@ -198,7 +198,7 @@ function sendConnectToAgent(instance: ExampleServiceDesk, connectMessage: Messag
       callback: (instance: ExampleServiceDesk) => {
         instance.callback.agentTyping(false);
         instance.callback.sendMessageToUser(
-          stringToMessageResponseFormat(getHelloMessage(instance)),
```

Comment:
You should remove the unused import for this.

Reply:
Unfortunately the other problem with this is that the TypeScript definition for `sendMessageToUser` doesn't allow for you to pass a string here. I was going to get around to it but in order to merge this PR, you'll need to make that change. You can update the function in `serviceDeskCallback.ts` and use this:
```
  /**
   * Sends a message to the chat widget from an agent. To display an error message to user, specify response_type:
   * 'inline_error' in MessageResponse, this also displays an error icon as well as hides the agent's avatar from user.
   *
   * @param message The message to display to the user. Before web chat 6.7.0, this function only supports providing a
   * MessageResponse; it does not support passing just a string.
   * @param agentID The ID of the agent who is sending the message.
   */
  sendMessageToUser(message: MessageResponse | string, agentID: string): void;
```

Reply:
Also removing the function has resulted in lint error (prettier) on each of these. Please make sure you have eslint enabled in your editor. Seems like we never added a `lint:fix` target to the `package.json`.

Reply:
@TazmanianDI, I've removed the import line from file.

I didn't understand the part you said about "it does not support passing just a string".

After removing the function I was able to receive everything just perfect, take a look:

![Captura de Tela 2022-10-20 às 10 02 37](https://user-images.githubusercontent.com/4713763/196956154-1aee69f5-e62d-4170-a748-208ece0c702e.png)

Do you want me to update the comment "_MessageResponse; it does not support passing just a string_" in the sendMessageToUser function?

BR,
Raffus

Reply:
Take a look at these tests:

![Captura de Tela 2022-10-20 às 10 06 03](https://user-images.githubusercontent.com/4713763/196956894-6bedc76d-6118-448d-b18f-b3b3544bc6c3.png)

![Captura de Tela 2022-10-20 às 10 06 11](https://user-images.githubusercontent.com/4713763/196956895-09e29c60-7036-4acd-93d6-7e04cf0a3678.png)

Cheers.

Reply:
The typescript problem is not a runtime problem, it is a build time problem. Your changes function correctly but they do not satisfy the type definitions we have in place for the library. Normally this should result in build errors but it seems we have a problem in our build system since it doesn't seem to be validating the typescript; that is something I need to investigate later. However, if you are using a typescript enabled editor it would show you the errors.
![image](https://user-images.githubusercontent.com/25828402/196963050-05096be5-84b1-4cbc-8faa-5b1f03086106.png)

I don't know if you are familiar with typescript or not, but we have type definitions that are intended to help ensure the right values are being passed around. If you look in the file I mentioned above (`serviceDeskCallback.ts`) you will see
```
sendMessageToUser(message: MessageResponse, agentID: string): void
```
What this means is that the `message` parameter must be of type `MessageResponse`. This is what `stringToMessageResponseFormat` returns. When you removed the `stringToMessageResponseFormat` that meant that the code was no longer passing a `MessageResponse` to `sendMessageToUser` but instead was passing a `string`. This is because prior to our most recent release of web chat this wasn't actually allowed. However it is now and we simply hadn't gotten around to updating the type definition. Where it says `message: MessageResponse` should be updated to say `message: MessageResponse | string`. This indicates that the message may be either a `MessageResponse` or a `string`. You just need to copy the update and the comment I pasted above into `serviceDeskCallback.ts`.

Reply:
I do also see that you removed the unnecessary import statement but there are still eslint errors in the file as well. They are basically just that now that you have shortened statements the linebreaks should be removed.
![image](https://user-images.githubusercontent.com/25828402/196964779-85bc9e34-3561-4dc1-b08d-50bac9ddf7a2.png)

Change:
```
        instance.callback.sendMessageToUser(
          getHelloMessage(instance),
          instance.agentProfile.id,
        );
```
to
```
        instance.callback.sendMessageToUser(getHelloMessage(instance), instance.agentProfile.id);
```
And same in the other cases.